deploy/20_operator.yaml

```diff
@@ -62,6 +62,7 @@ spec:
               cpu: "200m"
             limits:
               memory: "1G"
+              cpu: "300m"
```

Comment:
Why not just make this `200m` to match the requests? Based on the data you showed, it doesn't even get anywhere close to `200m`.

Reply:
thought about it. But I just like to over estimate a bit. Just in case. There is always edge cases or stuff that might happen that might spike things a bit.